nipoppy/cli.py

```diff
@@ -343,3 +344,81 @@ def status(**params):
     params = dep_params(**params)
     with handle_exception(StatusWorkflow(**params)) as workflow:
         workflow.run()
+
+
+@cli.group(cls=OrderedGroup)
+def pipeline():
+    """Local pipeline management."""
+    pass
+
+
+def zenodo_options(func):
+    """Define global options for the CLI."""
+    func = click.option(
+        "--zenodo-token",
+        envvar="ZENODO_TOKEN",
+        type=str,
+        required=False,
+        help="Zenodo access token.",
+    )(func)
+    func = click.option(
+        "--zenodo-api",
+        envvar="ZENODO_API",
+        default="https://zenodo.org/api",
+        type=str,
+        required=False,
+        help="Zenodo access token.",
+    )(func)
```

Comment:
@michellewang, do you think it would be better to have a `--sandbox` flag instead? It might be more intuitive for the users.